test/tracks.e2e-spec.ts

```diff
@@ -1,21 +1,61 @@
-import { INestApplication, ValidationPipe } from '@nestjs/common';
+import { Track } from './../src/tracks/models/track.entity';
+import { JwtPayload } from './../src/auth/models/jwt-payload';
+import { RolesGuard } from '../src/auth/guards/roles.guard';
```

Comment:
Codacy found an issue: ['RolesGuard' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5747194)